PBR/src/PBR_Renderer.cpp

```diff
@@ -621,8 +621,8 @@ void PBR_Renderer::PrecomputeCubemaps(IDeviceContext* pCtx,
         .Add("ENV_MAP_TYPE_SPHERE", static_cast<int>(IBL_PSOKey::ENV_MAP_TYPE_SPHERE))
         .Add("ENV_MAP_TYPE", static_cast<int>(EnvMapType));
 
-    auto& pPrecomputeIrradianceCubePSO = m_IBL_PSOCache[IBL_PSOKey{IBL_PSOKey::PSO_TYPE_IRRADIANCE_CUBE, EnvMapType, FeatureFlags, m_pIrradianceCubeSRV->GetDesc().Format}];
-    if (!pPrecomputeIrradianceCubePSO)
+    auto& pPrecomputeIrradianceCubeRenderTech = m_IBL_PSOCache[IBL_PSOKey{IBL_PSOKey::PSO_TYPE_IRRADIANCE_CUBE, EnvMapType, FeatureFlags, m_pIrradianceCubeSRV->GetDesc().Format}];
```

Comment:
Let's make it a bit shorter (besides, this is not a pointer anymore):
`PrecomputeIrradianceCubeTech`